### PR TITLE
Phase 1: ORCHIDE-aligned schema + policy tests + Kueue renderer

### PR DESCRIPTION
**Pull request by thc1006**
## Summary
- Align mission plan schema with ORCHIDE KubeCon EU 2026 slide 9 format (orbit, duration_seconds, landscape_type, model_validator for ACQ/DOWNLOAD constraints)
- Add Kueue Job renderer + `render-kueue` CLI subcommand
- Add 27 tests (schema, policy, Kueue, negative validation)
- Expand golden eval to 6-field comparison
- Remove dead `hera-workflows` dependency, fix `requires-python` to `>=3.10`
- Rewrite all 28 .md files with ORCHIDE-complementary positioning
- Add `docs/13_market_positioning.md` with cross-validated competitive analysis
- Add Claude Code agent hardening (settings.json, PostToolUse hook)

## Test plan
- [x] `python3 scripts/verify.py` — exit 0
- [x] `make test` — 27 passed
- [x] `make eval` — 2/2 passed
- [x] `make argo-smoke` — argo lint passed
- [x] `make opa-smoke` — OPA policy eval passed
- [x] `bash scripts/kueue_integration_smoke.sh` — Kueue admission confirmed
- [x] Existing sample plans backward-compatible

### CODE REVIEW

File: docs/04_architecture.md [hunk 1]
```diff
@@ -5,36 +5,92 @@ A **Mission Plan Compiler + Policy Guard + Admission Bridge + Workflow Renderer*
...
+Schema validation (Pydantic) and policy validation (OPA/Rego) intentionally overlap on some rules for defense-in-depth:
+
+| Constraint | Schema (schemas.py) | Policy (mission_plan.rego) |
+|---|---|---|
+| ACQ must have instrument | `model_validator` raises ValueError | — |
+| DOWNLOAD must not have services | `model_validator` raises ValueError | Rule 7 denies |
+| DOWNLOAD must have visibility | `model_validator` raises ValueError | Rule 8 denies |
+| Priority must not be 0 | `Field(ge=0)` allows it | Rule 5 denies it |
+| GPU+acceleration needs fallback | — | Rule 4 denies |
+| CPU+acceleration is contradictory | — | Rule 6 denies |
```
[the remark this answers is not in the document]
Comment (thc1006) [reply docs/04_architecture.md]: Fixed: table updated with Field(min_length=1) in schema column.

File: src/orbital_mission_compiler/cli.py [hunk 2]
```diff
@@ -52,6 +62,20 @@ def cmd_inspect(args):
...
+    plan = load_mission_plan(args.input)
+    intents = compile_plan_to_intents(plan)
+    out_dir = Path(args.output_dir)
+    out_dir.mkdir(parents=True, exist_ok=True)
+    written = []
+    for intent in intents:
+        job = render_kueue_job(intent, queue_name=args.queue, namespace=args.namespace)
+        out = out_dir / f"{intent.workflow_name[:50]}-kueue.yaml"
+        out.write_text(yaml.safe_dump(job, sort_keys=False), encoding="utf-8")
+        written.append(str(out))
```
[the remark this answers is not in the document]
Comment (thc1006) [reply src/orbital_mission_compiler/cli.py]: Fixed: output filename now sanitized via _sanitize_k8s_name() to prevent path traversal.
